hive-metastore/src/main/java/org/apache/iceberg/hive/MetastoreUtil.java

```diff
@@ -54,8 +54,8 @@ public static boolean hive3PresentOnClasspath() {
   }
 
   /**
-   * Invokes alterTable via metaStoreClient without triggering recursive listing.
-   * ref: https://github.com/apache/iceberg/pull/734#issuecomment-574062407
+   * If possible, call the alter_table method via metaStoreClient with an environment context that turns off stats
```

Comment:
> Nits: Why "if possible"? The final comma is also not needed so

It's due to this signature which is added as final fallback 
```java
      .impl(IMetaStoreClient.class, "alter_table",
          String.class, String.class, Table.class)
```

> "The final comma is also not needed so" 

makes sense.

should I rephrase it to : 

" If possible, calls alter_table method using the metastore client with an environmental context that turns off stats update to avoid recursive file listing"

Reply:
The "if possible" here applies to the whole sentence which makes it sound like this may call alter_table or it may not. So maybe instead
"Calls alter_table method using the metastore client. If possible an environmental context will be used that ...."